pages/instructorRequestCourse/instructorRequestCourse.ejs

```diff
@@ -78,10 +88,39 @@
                                 Providing your GitHub username will allow you to edit course content offline.  You do not need to provide this if you would like to use the online web editor.
                             </small>
                         </div>
+                        <div class="form-group">
+                            <label>Your Role in the Course</label><br>
+                            <input type="radio" id="role-instructor" name="cr-role" value="instructor">
+                            <label for="role-instructor">Official Course Instructor</label><br>
```

Comment:
Rather than using `<br>` here, I think bootstrap recommends the use of the `form-check` class on divs around each radio option: https://getbootstrap.com/docs/4.5/components/forms/#checkboxes-and-radios